lib/committee/parameter_coercer.rb

```diff
@@ -5,31 +5,63 @@ def initialize(params, schema, options = {})
       @schema = schema
 
       @coerce_date_times = options.fetch(:coerce_date_times, false)
+      @coerce_recursive = options.fetch(:coerce_recursive, false)
     end
 
-    def call
-      coerced = {}
-      return coerced unless @schema.respond_to?(:properties)
+    def call!
+      coercer_object!(@params, @schema)
+    end
+
+    private
 
-      @schema.properties.each do |k, s|
-        original_val = @params[k]
+      def coercer_object!(hash, schema)
```

Comment:
Nice use of recursion all over here!

Do you think that we could rename each of these methods to `coerce_*` instead of `coercer_*`?